api/add_retention_policy.md

```diff
@@ -61,5 +62,11 @@ Create a data retention policy with an integer-based time column:
 SELECT add_retention_policy('conditions', BIGINT '600000');
 ```
 
+Create a data retention policy to discard chunks created before 6 months:
+
+```sql
+SELECT add_retention_policy('conditions', drop_created_before => INTERVAL '6 months');
+```
+
```

Comment:
You should probably elaborate a little here as well. Give the chunk interval for the table and explain what range of data that will be dropped.

Reply:
Again, chunk interval and creation times are not related. In events use cases where serial id or integer-like ranges are used, we use the creation time metadata where applicable.

Reply:
I still think some sort of example here might make the situation a little clearer, but it is not a blocker.